File: src/Configuration/ConfigFileScope.cs
/// <summary>
/// Defines the scope of configuration settings.
/// </summary>
public enum ConfigFileScope
{
    /// <summary>
    /// Global settings (system-wide).
    /// </summary>
    Global,

    /// <summary>
    /// User settings (user-specific).
    /// </summary>
    User,

    /// <summary>
    /// Local settings (project-specific).
    /// </summary>
    Local,
    
    /// <summary>
    /// All scopes (used for display/querying across scopes).
    /// </summary>
    Any
}

File: ts/src/Configuration/ConfigFileScope.ts
/**
 * Defines the scope of configuration settings.
 */
export enum ConfigFileScope {
  /**
   * Global settings (system-wide).
   */
  Global = 'global',

  /**
   * User settings (user-specific).
   */
  User = 'user',

  /**
   * Local settings (project-specific).
   */
  Local = 'local',

  /**
   * Config file settings (from explicitly loaded configuration file paths).
   */
  FileName = 'filename',

  /**
   * All scopes (used for display/querying across scopes).
   */
  Any = 'any',
}